### ISSUE TITLE
REQUEST: dark menu bar yosemite, matching menu bar icon

### ISSUE THREAD

**Issue by GoogleCodeExporter** [irishuang1211/macosx-nosleep-extension#142]
```
To add additional menu bar icon to match the new dark menu bar theme in Yosemite

nosleep 1.3.3, Mac 10.10
```

Original issue reported on code.google.com by `lightlov...@gmail.com` on 27 Oct 2014 at 3:41

**Comment by GoogleCodeExporter** [comment]
```
Implemented in 1.4.0
```

Original comment by `integral...@gmail.com` on 15 Nov 2014 at 7:18

**Comment by GoogleCodeExporter** [comment]
Original comment by `integral...@gmail.com` on 15 Nov 2014 at 7:18
- Changed state: **Fixed**
